docs/release-notes/release-notes.md

```diff
@@ -2,17 +2,32 @@
 
 This guide provides notes for major version releases. These notes may be helpful for users when upgrading from previous major versions.
 
+## v3.0.0
+
+### Node Operators (v3.0.0)
+
+- Consensus node operators must enable the BBR (Bottleneck Bandwidth and Round-trip propagation time) congestion control algorithm. See [#3774](https://github.com/celestiaorg/celestia-app/pull/3774).
+- Consensus node operators should manually configure their node's mempool `ttl-num-blocks = 12` in config.toml. An example command to do this:
+
+  ```bash
+  sed -i 's/ttl-num-blocks = 5/ttl-num-blocks = 12/' ~/.celestia-app/config/config.toml
+  ```
```

Comment:
Does this need to be indented?

Reply:
I think desirable for the code block to be indented so it appears under the bullet point that it's related to

![Screenshot 2024-10-22 at 11 58 22 AM](https://github.com/user-attachments/assets/12f47796-67eb-4c4b-92e4-fcfc9f0ef9a6)